### ISSUE TITLE
Please update documentation

### ISSUE THREAD

**Issue by sethlinnkuleuven** [simplesamlphp/simplesamlphp-module-discopower#19]
The documentation says to add this to authsources.php
<?php
$config = [
    'default-sp' => [
        'saml:SP',
        'discoURL' => 'https://sp1.example.org/simplesaml/module.php/discopower/disco.php',
    ],
];

but disco.php was removed in v1.2.0  I also tried to use module.php/discopower/src/Controller/DiscoPower.php
but that didn't work.

Thank you

**Comment by tvdijen** [comment]
/disco.php should still work, even without the file in place. What is the issue?

**Comment by sethlinnkuleuven** [comment]
In the UI when I go to test -> default-sp the URL is 
https://example.com/ssp/module.php/saml/disco?entityID=......
Isn't this the default disco service. How do I kno if I am actually using discopower and not eh built in. FYI: in the UI it says that discopower is enabled.

**Comment by thijskh** [comment]
It should look like this to use discopower:

```php
    'default-sp' => [
        'saml:SP',

        // The entity ID of this SP.
        // Can be NULL/unset, in which case an entity ID is generated based on the metadata URL.
        'entityID' => ...

        // The entity ID of the IdP this SP should contact.
        // Can be NULL/unset, in which case the user will be shown a list of available IdPs.
        'idp' => null,

        // The URL to the discovery service.
        // Can be NULL/unset, in which case a builtin discovery service will be used.
        'discoURL' => '/ssp/module.php/discopower/disco.php',
```

Maybe you can show your `authsources.php` to see what you have there exactly?

**Comment by sethlinnkuleuven** [comment]
I am very sorry, I miss spelled discoURL.  It seems to be working fine.  If you don't mind  me asking one more stupid question about discopower....
I am using the eduGAIN federating and in my saml20-idp-remote.php file I have this

$metadata['https://sts.windows.net/6397936e-86f3-4537-bbb4-927438c12ab3/'] = array (
  'entityid' => 'https://sts.windows.net/6397936e-86f3-4537-bbb4-927438c12ab3/',
  'description' => 
  array (
    'nl' => 'Vlir',
  ),
  'OrganizationName' => 
  array (
    'nl' => 'Vlir',
  ),
  'name' => 
  array (
    'nl' => 'Vlir',
  ), ......

for some reason it does not recognize the 'nl' and defaults to to the entityid. This is not a problem if the language is 'en'
in other words, this does not work
  'name' => 
  array (
    'nl' => 'Vlir',
  ),
but this does
  'name' => 
  array (
    'en' => 'Vlir',
  ),

**Comment by thijskh** [comment]
SimpleSAMLphp will pick the name of the current language of the user, that is, if your UI is in Dutch then it will/should show the `nl` name. If your UI is not in Dutch and the metadata does not provide that language, it will fall back to checking the default. Whatever the default is, is defined in `config.php` in `languages.default` (defaults to English).

Long story short, you can change `languages.default` to `nl` for this specific case, or, probably more compatible in general, have the IdP also add an `en` name variant.
